doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_qsg.rst

```diff
@@ -75,22 +75,29 @@ For this quick start guide, we will install the following software:
 Building the SUIT sample
 ************************
 
-Start by building the SUIT sample:
+Start by building the :ref:`nrf54h_suit_sample` sample:
 
 .. code-block:: console
 
-   west build -b nrf54h20dk/nrf54h20/cpuapp nrf/samples/suit/smp_transfer
+   west build --sysbuild -b nrf54h20dk/nrf54h20/cpuapp nrf/samples/suit/smp_transfer
 
-This command builds the SUIT sample for the nRF54H20 SoC.
+This command builds the :ref:`nrf54h_suit_sample` sample for the nRF54H20 SoC.
 
 Modifying class and vendor identifiers
 **************************************
 
-The next step involves customizing identifiers in the manifest:
+Replacing and using the correct class and vendor UUIDs prevents conflicts in the DFU process.
```

Comment:
Can we try to elaborate more on it? That it assures that only firmware designed for this type of devices will be updated or sth like that?

Reply:
Ok, added an additional sentence with further explanation.